backend/src/main/java/com/daedan/festabook/global/logging/LoggingAspect.java

```diff
@@ -29,22 +36,26 @@ public class LoggingAspect {
     )
     public Object allLayersLogging(ProceedingJoinPoint joinPoint) throws Throwable {
         StopWatch stopWatch = new StopWatch();
-        String className = joinPoint.getSignature().getDeclaringTypeName();
+        String className = joinPoint.getSignature().getDeclaringType().getSimpleName();
         String methodName = joinPoint.getSignature().getName();
 
         MethodEventLog methodEvent = MethodEventLog.from(className, methodName);
         log.info("", kv("event", methodEvent));
 
+        Span span = tracer.spanBuilder(className + "::" + methodName).startSpan();
```

Comment:
tracer.spanBuilder에 무엇이 들어가는지 설명하기 위해 변수로 빼도 좋을 것 같아요.

```suggestion
        String spanName = "className + "::" + methodName";
        Span span = tracer.spanBuilder(spanName).startSpan();
```